### DISCUSSION TITLE
How to change the "news" section header to "Accouncements" ?

### DISCUSSION THREAD

**Discussion by anubhabdaserrr** [alshedivat/al-folio#1982] [Q&A] [answered]
For the custom sections, the title attribute can be changed for this purpose. Additionally, the blog also has a blog_nav_title attribute which can be changed. Is there a news_title attribute that be changed?

**Comment by george-gca** [comment, marked as answer]
You can change it in [_layouts/about.html](https://github.com/alshedivat/al-folio/blob/f9b8d450813cc36bf783860b806e3edcb27a4418/_layouts/about.html#L46).
